Bixal/move.mil
# fixed post-deploy script to deploy to aws correctly

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Fixes a syntax error that prevented this eb extension from deploying with a previous version of the eb extension 

## Testing

To verify the changes proposed in this pull request…

1. deploy this to staging
2. deploy another change that would require a cache reset, menu update, etc
3. check logs that the script ran or simply that the second change correctly deployed and updated
